src/IO/S3/PocoHTTPClient.cpp

```diff
@@ -50,9 +50,11 @@ namespace DB::S3
 
 PocoHTTPClientConfiguration::PocoHTTPClientConfiguration(
         const Aws::Client::ClientConfiguration & cfg,
-        const RemoteHostFilter & remote_host_filter_)
+        const RemoteHostFilter & remote_host_filter_,
+        unsigned int s3_max_redirects_)
```

Comment:
unsigned int -- quite strange to use this type, but ok, doesn't affect anything.